application/__init__.py

```diff
@@ -54,11 +62,13 @@
 profiles: Profile
 if os.getenv('DBADDR') is not None:
     from .Profile.DBProfile import DBProfile
+
     profiles = DBProfile(app)
 else:
     from .Profile.LocalProfile import LocalProfile
+
     profiles = LocalProfile()
 
 api = Blueprint('api', __name__)
 
-from .routes import *
+from .routes import *
```

Comment:
Add EOL (\n) at the end of the file.
```suggestion
from .routes import *

```